Index: app/src/main/java/com/maths/beyond_school_280720220930/subjects/Maths.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.maths.beyond_school_280720220930.subjects;\r\n\r\npublic class Maths {\r\n\r\n}\r\n
===================================================================
diff --git a/app/src/main/java/com/maths/beyond_school_280720220930/subjects/Maths.java b/app/src/main/java/com/maths/beyond_school_280720220930/subjects/Maths.java
--- a/app/src/main/java/com/maths/beyond_school_280720220930/subjects/Maths.java	
+++ b/app/src/main/java/com/maths/beyond_school_280720220930/subjects/Maths.java	
@@ -2,4 +2,14 @@
 
 public class Maths {
 
+
+
+    public String generateMultiplicationQuestion(){
+
+
+        String  question="2 2 s are";
+
+        return "";
+    }
+
 }
